components/content/index.css

```
@@ -1,21 +1,42 @@
 .content {
     background-color: var(--blueviolet);
+    border-radius: 12px;
 }
 
 .content-holder {
     display: flex;
     height: 100%;
-    background-image: linear-gradient(var(--red), var(--green));
+    border-radius: 12px;
 }
 
 /* Content aside */
 
 .content-aside {
-    width: 20%;
+    width: 40%;
     min-width: 100px;
-    background-image: url(../../assets/img/aside-background.jpg);
+    background-color: var( --light-gray-tint-1 );
```

```suggestion
    background-color: var( --light-gray-tint-0 );
```

Please, add new color to our colors list `--light-gray-tint-0: #F9F9F9;`